st2actions/st2actions/query/mistral/v2.py

```diff
@@ -129,8 +129,8 @@ def _format_task_result(self, task):
 
         result['id'] = task['id']
         result['name'] = task['name']
-        result['execution_id'] = task.get('execution_id', None)
-        result['workflow'] = task['wf_name']
+        result['workflow_execution_id'] = task.get('workflow_execution_id', None)
```

Comment:
To be honest though, I like the new attribute names more since they are more explicit - so this time, we should probably change the names on our side as well (but in the future, if needed, we should consider the other option) :)